src/node.cc

```diff
@@ -3398,6 +3402,8 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--v8-options") == 0) {
       new_v8_argv[new_v8_argc] = "--help";
       new_v8_argc += 1;
+    } else if (strncmp(arg, "--v8-pool-size=", 15) == 0) {
+      v8_thread_pool_size = arg + 15;
```

Comment:
@tomgco ... essentially, rather than defining `v8_thread_pool_size` as a `const char*`, you can create it as an int with a default value of `v8_default_thread_pool_size`, then, in the check for `--v8-pool-size`, parse the string immediately, if it's within the appropriate int range ( > 0, <= the current maximum ... which is 4) set `v8_thread_pool_size` to the new value. You can then avoid the `pool_size` declaration later and simply `default_platform = v8::platform::CreateDefaultPlatform(v8_thread_pool_size)`.